### ISSUE TITLE
Slimefun_Not_Working_As_Intended

### ISSUE THREAD

**Issue by PeculiarLamp** [Slimefun/Slimefun4#3874]
### ❗ Checklist

- [X] I am using the official english version of Slimefun and did not modify the jar.
- [X] I am using an up to date "DEV" (not "RC") version of Slimefun.
- [X] I am aware that issues related to Slimefun addons need to be reported on their bug trackers and not here.
- [X] I searched for similar open issues and could not find an existing bug report on this.

### 📍 Description

When a member logged on and messed with the Slimefun components, one output chest was working while the other was not and when he tried to use an energy regulator it did not show him as owner rather it shown CS-CoreLib as owner and when he tried to destroy it and place it back it just shown it as CS-CoreLib's head rather than an actual component including the "Connecting..." hologram as well as the Simple Elevator when trying to be used it doesn't work and when destroyed and put into inventory it says Block of Quartz. After these problems rose, I saw in the console a message that states "[20:15:08 ERROR]: [Slimefun] An Exception was caught while ticking the Block Tickers Task for Slimefun vDEV - 1068 (git cab97b1a)
java.lang.IllegalStateException: World "Teknocraft S3.5" seems to have been deleted. Do not call unsafe methods directly!" Even though in the world settings folder of Slimefun has the world on it. The software used is a mixture of different softwares combined: Spigot, Paper, Purpur and Bukkit.

### 📑 Reproduction Steps

1. Updated  to the latest version
2. Uploaded it to my server's file manager
3. Restarted the server
4. Logged on and the issues still occured

### 💡 Expected Behavior

Was expecting for everything to work as normal after these issues, but alas, they still remain.

### 📷 Screenshots / Videos

![SimpleElevatorIssue](https://github.com/Slimefun/Slimefun4/assets/137963112/b4b54253-2bf4-4dda-9958-6d2245470dea)
![WorldSettingsSlimefun](https://github.com/Slimefun/Slimefun4/assets/137963112/4cb03b5f-9935-4411-9eb2-10a12e4b59ad)
![CoreLib'sHead](https://github.com/Slimefun/Slimefun4/assets/137963112/38d67653-72d4-4a82-970d-d39678323385)
![AnExceptionWasCaught-Slimefun](https://github.com/Slimefun/Slimefun4/assets/137963112/83abee3b-17c0-4401-b193-cbc61c85283e)
![Owner_CSCoreLib](https://github.com/Slimefun/Slimefun4/assets/137963112/ea9b4ff2-c977-488e-b173-9e7cdffabcc0)

### 💻 Server Software

Other (please specify in your description)

### 🎮 Minecraft Version

1.19.x

### ⭐ Slimefun version

Slimefun4 - DEV 1068 (git cab97)

### 🧭 Other plugins

SimpleUtilities

**Comment by JustAHuman-xD** [comment]
This is a duplicate, you're blockstorage got messed up that's all

**Comment by JustAHuman-xD** [comment]
Sorry not a duplicate I'm blind

**Comment by JustAHuman-xD** [comment]
The world thing is an interesting problem.

**Comment by JustAHuman-xD** [comment]
Can you provide the logs like the issue template asked for?

**Comment by Boomer-1** [comment]
also what do you mean a mixture of 4 different servers? what type of server are you using, as slimefun isn't compatible with bukkit

**Comment by PeculiarLamp** [comment]
> Can you provide the logs like the issue template asked for?

[2023-03-17-1.log.gz](https://github.com/Slimefun/Slimefun4/files/11888450/2023-03-17-1.log.gz)

**Comment by JustAHuman-xD** [comment]
and the error reports? (If there are any)

**Comment by PeculiarLamp** [comment]
> also what do you mean a mixture of 4 different servers? what type of server are you using, as slimefun isn't compatible with bukkit

It isn't 4 different servers it is 4 different softwares working on one server, though after you said that Slimefun isn't compatible with Bukkit, I deleted Bukkit off and the issues still prevailed.

**Comment by Boomer-1** [comment]
that log has nothing in it to indicate any issues. 

please provide the full screenshot of /sf versions

**Comment by PeculiarLamp** [comment]
> and the error reports? (If there are any)
![EmptyDirectory](https://github.com/Slimefun/Slimefun4/assets/137963112/b7e1c687-2127-4209-a805-16cc4363214d)

The error reports are empty on my end.

**Comment by PeculiarLamp** [comment]
> that log has nothing in it to indicate any issues.
> 
> please provide the full screenshot of /sf versions

I apologize, I sent you an older log, anyway, here is a more recent one.
[2023-06-26-1.log.gz](https://github.com/Slimefun/Slimefun4/files/11888521/2023-06-26-1.log.gz)

**Comment by PeculiarLamp** [comment]
> that log has nothing in it to indicate any issues.
> 
> please provide the full screenshot of /sf versions

And, here are the SF Versions
![SFVersions](https://github.com/Slimefun/Slimefun4/assets/137963112/8b7eab06-1125-4e9c-b0ee-bc4c811e71ba)

**Comment by PeculiarLamp** [comment]
> can you please be specific on what you mean by you're using 4 different types of software? you're using a purpur server, but what do you mean by a combination of 4 different software?

My server uses the following software: Bukkit, Paper, Purpur and Spigot

**Comment by Boomer-1** [comment]
what do you mean, you have a purpur server. please explain in detail

**Comment by ybw0014** [comment]
Do not use weird symbols other than underscores(_) in world name.

**Comment by JustAHuman-xD** [comment]
Yeah how do you use multiple softwares on 1 server? I've never heard of that and I can't even think about how it would work as they're _separate_ jars. And running them at the same time would just be different servers that maybe share some files? That'd probably error.

**Comment by JustAHuman-xD** [comment]
> Do not use weird symbols other than underscores(_) in world name.

This too, I've heard using `.` and other symbols *breaks* things

**Comment by PeculiarLamp** [comment]
> > Do not use weird symbols other than underscores(_) in world name.
> 
> This too, I've heard using `.` and other symbols _breaks_ things

How to rename the world then?

**Comment by PeculiarLamp** [comment]
> > > Do not use weird symbols other than underscores(_) in world name.
> > 
> > 
> > This too, I've heard using `.` and other symbols _breaks_ things
> 
> How to rename the world then?

And still keep everything.

**Comment by ybw0014** [comment]
> Yeah how do you use multiple softwares on 1 server? I've never heard of that and I can't even think about how it would work as they're _separate_ jars. And running them at the same time would just be different servers that maybe share some files? That'd probably error.

Maybe he doesn't know relationship among bukkit, spigot, paper and purpur. It is only purpur.

**Comment by J3fftw1** [comment]
Slimefun should just straight up not run if there are illegal charachters in the world name.
We might just need to add an check to stop Slimefun for running at all.
So for what ever reason it does work sometimes it doesnt let you make Slimefun work.

**Comment by J3fftw1** [comment]
> > > > Do not use weird symbols other than underscores(_) in world name.
> > > 
> > > 
> > > This too, I've heard using `.` and other symbols _breaks_ things
> > 
> > 
> > How to rename the world then?
> 
> And still keep everything.

i dont think you can rename the world easily.
I would back it up first.
Then if you have multiverse core you can do /mvm set alias (NAME). run that command when you are in the world.

**Comment by PeculiarLamp** [comment]
> > > > > Do not use weird symbols other than underscores(_) in world name.
> > > > 
> > > > 
> > > > This too, I've heard using `.` and other symbols _breaks_ things
> > > 
> > > 
> > > How to rename the world then?
> > 
> > 
> > And still keep everything.
> 
> i dont think you can rename the world easily. I would back it up first. Then if you have multiverse core you can do /mvm set alias (NAME). run that command when you are in the world.

Okay, might try that, thanks!

**Comment by Boomer-1** [comment]
because this was the result of incorrectly naming a world should this be closed?
